### PR TITLE
Propose initial commandline instruction

### PR DESCRIPTION
**Pull request by mbax**
I'm legitimately unsure how this will even render. But, it's my start.

### CODE REVIEW

File: source/server/getting-started/installation.rst [hunk 1]
```diff
@@ -45,18 +45,38 @@ Dedicated Servers
...
     version if it's older than the one that :doc:`sponge requires <implementations/spongeforge>`.
 
-Via GUI
---------
+Installing Forge via Commandline
+--------------------------------
+
+1. Visit the `Minecraft Forge website <http://files.minecraftforge.net/>`_ and click "Show all downloads" to view the full
+   set of available options. Identify the latest version, or any version that's newer than the one listed :doc:`in the
+   filename of the Sponge download <implementations/spongeforge>`, and hover over the (i) next to "Installer" to get the
```
Comment (gabizou) [inline source/server/getting-started/installation.rst]: SpongeForge, not Sponge ;)

File: source/server/getting-started/installation.rst [hunk 2]
```diff
@@ -45,18 +45,38 @@ Dedicated Servers
     If you already have a Forge server, just put the Sponge mod into your ``mods`` folder. Remember to update your Forge
     version if it's older than the one that :doc:`sponge requires <implementations/spongeforge>`.
 
-Via GUI
---------
+Installing Forge via Commandline
+--------------------------------
+
+1. Visit the `Minecraft Forge website <http://files.minecraftforge.net/>`_ and click "Show all downloads" to view the full
+   set of available options. Identify the latest version, or any version that's newer than the one listed :doc:`in the
```
Comment (progwml6) [inline source/server/getting-started/installation.rst]: forge version should match the jar name

Comment (mbax) [inline source/server/getting-started/installation.rst]: That bit about "newer than" is copypasted from the existing content. I'll update the page.

File: source/server/getting-started/installation.rst (hunk 1 above)
Comment (mbax) [inline source/server/getting-started/installation.rst]: Copied from existing content, will update page

File: source/server/getting-started/installation.rst [hunk 3]
```diff
@@ -43,20 +43,38 @@ Dedicated Servers
...
+#. From the folder in which you wish to install Forge, execute the jar with the ``--installServer`` option. Example:
+   ``java -jar forge-version-installer.jar --installServer``
+#.  Continue to Adding SpongeForge to Forge below.
+
+
+Installing Forge via GUI
+------------------------
+
+1. Download the Minecraft Forge installer from the `Minecraft Forge website <http://files.minecraftforge.net/>` for the version
+   matching the one listed :doc:`in the filename of the SpongeForge download <implementations/spongeforge>`_.
```
Comment (Inscrutable) [inline source/server/getting-started/installation.rst]: I think this is what Travis is grumpy about. The stray underscore after the :doc: link breaks it.

**Comment by mbax** [comment]
Thanks

**Pull request merged**